docs/API.md

```diff
@@ -715,29 +721,23 @@ Example response:
 
 An object with the following properties:
 
-* `"offset"`: An integer, optional. (default: 0).
-* `"limit"`: An integer, optional. (default: 200).
+* `"offset"`: An integer, optional. (default: 0). Position of the first returned element from the database returned list.  
+* `"limit"`: An integer, optional. (default: 200). Number of element returned from the *offset* element.
 * `"frontend_params"`: As described below.
 
-The value of `"frontend_params"` is an object in turn, with the
-keys `"domain"` and `"nameservers"`. `"domain"` and `"nameservers"`
-will be used to look up all tests for the given domain, separated
-according to if they were started with a `"nameservers"` parameter or
-not.
-
->
-> TODO: Do we have an SQL injection opportunity here?
->
-> TODO: Describe the remaining keys in the example
->
-> TODO: Describe the purpose of `"offset"` and `"limit"`
->
-> TODO: Is the `"nameservers"` value a boolean in disguise?
->
-> TODO: The description of `"frontend_params"` is clearly not up to date. Can it
-> be described in a better way?
->
-
+* `"client_id"`: A free-form string, optional.
```

Comment:
Please add this paragraph before the "`frontend_params`" properties list:

> The value of "`frontend_params`" is an object with the following properties:

Also the "`domain`" parameter seems to have gone missing.